locale/zh_CN.po

```diff
@@ -6602,17 +6603,17 @@ msgstr "工具提示文字"
 #. TRANSLATORS: Keyword of system colour
 #: ../src/propgrid/advprops.cpp:795
 msgid "Window"
-msgstr "视窗"
+msgstr "视窗(&W)"
 
 #. TRANSLATORS: Keyword of system colour
 #: ../src/propgrid/advprops.cpp:796
 msgid "WindowFrame"
-msgstr "窗框"
+msgstr "窗框(&W)"
 
 #. TRANSLATORS: Keyword of system colour
 #: ../src/propgrid/advprops.cpp:797
 msgid "WindowText"
-msgstr "视窗文字"
+msgstr "视窗文字(&W)"
```

Comment:
No keyboard shortcut in the original English message.

Reply:
I suppose a full check is required, the later the better.
![Screen Shot 2022-07-06 at 2 18 24 AM](https://user-images.githubusercontent.com/82932766/177390937-228d4958-886b-4bcf-a212-a6abb43ab289.png)